uki-generate

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3 -I
+import os
+import rpm
+import subprocess
+import sys
+from typing import List, Tuple
+
+def main(args):
+    alignment_mask = (1 << 21) - 1
+    if len(args) != 7 or args[1] != '--':
+        print(f"Usage: uki-generate -- HYPERVISOR CONFIG KERNEL INITRAMFS OUTPUT", file=sys.stderr)
```

Comment:
Please drop requirement of `--` (here, and generally in future tools you create). In this case, the tool doesn't even support any kind of options!

Reply:
Ack.